Gulpfile.js

```diff
@@ -19,6 +20,13 @@ gulp.task('serve', ['sass', 'js'], function() {
     gulp.watch('public/*.html').on('change', browserSync.reload);
 });
 
+// Run lint for js
+gulp.task('lint', function() {
```

Comment:
Yea, probably better. I wanted to wait until we got sass linter up to and just add both of them, but I don't think it matters